sdk/face/azure-ai-face/azure/ai/face/aio/operations/_operations.py

```diff
@@ -0,0 +1,15379 @@
+                deserialization_callback=get_long_running_output,
+            )
+        return AsyncLROPoller[_models.DynamicPersonGroup](
+            self._client, raw_result, get_long_running_output, polling_method  # type: ignore
+        )
+
+    async def _delete_group_initial(  # pylint: disable=inconsistent-return-statements
+        self, dynamic_person_group_id: str, **kwargs: Any
+    ) -> None:
+        error_map = {
+            401: ClientAuthenticationError,
+            404: ResourceNotFoundError,
+            409: ResourceExistsError,
+            304: ResourceNotModifiedError,
+        }
+        error_map.update(kwargs.pop("error_map", {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls: ClsType[None] = kwargs.pop("cls", None)
+
+        _request = build_person_directory_dynamic_person_groups_delete_group_request(
+            dynamic_person_group_id=dynamic_person_group_id,
+            api_version=self._config.api_version,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
+        }
+        _request.url = self._client.format_url(_request.url, **path_format_arguments)
+
+        _stream = False
+        pipeline_response: PipelineResponse = await self._client._pipeline.run(  # pylint: disable=protected-access
+            _request, stream=_stream, **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [202]:
+            if _stream:
+                await response.read()  # Load the body in memory and close the socket
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        response_headers = {}
+        response_headers["operation-Location"] = self._deserialize("str", response.headers.get("operation-Location"))
+
+        if cls:
+            return cls(pipeline_response, None, response_headers)  # type: ignore
+
+    @distributed_trace_async
+    async def begin_delete_group(
+        self, dynamic_person_group_id: str, **kwargs: Any
+    ) -> AsyncLROPoller[_models.AsyncOperationStatus]:
+        # pylint: disable=line-too-long
+        """The most basic operation that applies to a resource.
+
+        :param dynamic_person_group_id: ID of the dynamic person group. Required.
+        :type dynamic_person_group_id: str
+        :return: An instance of AsyncLROPoller that returns AsyncOperationStatus. The
+         AsyncOperationStatus is compatible with MutableMapping
+        :rtype: ~azure.core.polling.AsyncLROPoller[~azure.ai.face.models.AsyncOperationStatus]
+        :raises ~azure.core.exceptions.HttpResponseError:
+
+        Example:
+            .. code-block:: python

```

Comment:
The `identify` methods don't need the suffix for clarification, as that's implied by their grouping. I would probably just call them all `idenfity_faces` or something similar.